build.gradle

```diff
@@ -0,0 +1,30 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.mparticle:android-kit-plugin:+'
+    }
+}
+
+apply plugin: 'com.mparticle.kit'
+
+repositories {
+    maven {
+        url 'https://artifactory.roko.mobi/artifactory/libs-release'
+    }
+}
+
+android {
+    defaultConfig {
+        minSdkVersion 16
+    }
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:24.2.1'
+    compile 'com.android.support:recyclerview-v7:24.2.1'
+    compile 'com.rokolabs:rokomobi-android-sdk:1.12.+@aar'
+    compile 'com.google.code.gson:gson:2.7'
```

Comment:
Is GSON necessary? This may be a major conflict with implementing customers, if they already have GSON in their app.

Reply:
yes, GSON is required for full Instabot functionality, @samdozor, how do typically handle these situations?

Reply:
I think in that case we just need to document the versions of GSON that your SDK is compatible with. That way an app developer can include their own version if they choose to, and then they can do the following when including the instabot-kit:

```groovy
//example build.gradle
compile('com.mparticle:android-instabot-kit:4.16.0') {
        exclude group: 'com.google.code.gson'
}
//include your custom version
compile 'com.google.code.gson:gson:2.8'
```

Can you comment with which versions of GSON you support?

Reply:
@samdozor  - we support up to, and including `com.google.code.gson:gson:2.7`

Reply:
ok cool. we'll document 2.0-2.7